bulletroboy/operator.py

```diff
@@ -76,7 +76,7 @@ def publish_state(self, ef_names = np.array(['human/left_hand','human/right_hand
 
 
 class Movements():
-	def __init__(self, operator, link = b'human/left_hand'):
+	def __init__(self, operator, link = b'left_wrist'):
```

Comment:
the whole point of parameters is that you dont need to change the funtion implementation when some high level things are changed, like the urdf model...

Reply:
This would be handled in the upcoming utils/constants file where links (human + roboy) would be "standardized"

Reply:
so i dont want to / not planning to review the same code multiple times. if you have a new version, where is it?